#! /bin/sh /usr/share/dpatch/dpatch-run
## man-wvware-options.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: help/man/wvWare.1: Document all options (#200776, #334769).

@DPATCH@
diff -urNad wv-1.2.1~/help/man/wvWare.1 wv-1.2.1/help/man/wvWare.1
--- wv-1.2.1~/help/man/wvWare.1	2004-08-07 22:44:12.000000000 +0200
+++ wv-1.2.1/help/man/wvWare.1	2006-03-18 15:06:09.000000000 +0100
@@ -41,10 +41,43 @@
 If dir is used then any created graphics will be placed in this directory. The default is the directory
 in which the word document resides.
 .TP
+.I "\-b name, \-\-basename name"
+Use
+.I name
+as the base name for image files.
+.TP
+.I "\-a format, \-\-auto-eps format"
+Depending on whether images of type
+.I format
+can be automatically converted to encapsulated PostScript, print "yes" or "no"
+and exit.  Encapsulated PostScript is required for LaTeX, PostScript, and PDF
+output.  Currently possible values for
+.I format
+are "wmf", "png", and "jpg".
+.TP
+.I "\-s format, \-\-suppress format"
+Suppress automatic conversion of images of type
+.I format
+to encapsulated PostScript.  Currently possible values for
+.I format
+are "wmf", "emf", "png", "jpg", and "pict".
+.TP
+.I "\-1, \-\-nographics"
+Suppress output of embedded images.
+.TP
 .I "\-x config.xml, \-\-config config.xml"
 Use this to give the location of the config file. If not used, the config file is searched for
 by first checking the current directory, and then by looking in the install location
 (/usr/local/lib/wv/wvHtml.xml by default)
+.TP
+.I "\-X, \-\-xml"
+Produce XHTML-like XML output (as defined in /usr/local/lib/wv/wvXml.xml).
+.TP
+.I "\-v, \-\-version"
+Print wvWare's version number and exit.
+.TP
+.I "\-?, \-\-help"
+Print a help message and exit.
 .SH MORE INFORMATION
 http://wvware.sourceforge.net
 .SH "SEE ALSO"
